#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
PodHeader pod_header
uint8 pod_state
bool door_open
bool charge_door_open
bool front_bumper
bool rear_bumper
uint8 driver_seat
uint8 passenger_seat
uint8 motor_temp
uint8 inverter_temp
uint16 vbat12_level